src/core/session.js

```diff
@@ -40,10 +40,12 @@ export class Session {
   progressBarDelay = 500
   started = false
   formMode = "on"
+  #pageRefreshDebouncePeriod = 150
 
   constructor(recentRequests) {
     this.recentRequests = recentRequests
     this.preloader = new Preloader(this, this.view.snapshotCache)
+    this.refresh = debounce(this.refresh.bind(this), this.pageRefreshDebouncePeriod)
```

Comment:
You could do `this.pageRefreshDebouncePeriod = this.pageRefreshDebouncePeriod` to reuse the refresh assignment.